docs/adr/0004-running-out-of-rancher-manager-cluster.md

```diff
@@ -0,0 +1,31 @@
+# 4. Running out of Rancher Manager cluster
+
+* Status: proposed
+* Date: 2023-08-31
+* Authors: @salasberryfin
+* Deciders: @richardcase @alexander-demicev @furkatgofurov7 @Danil-Grigorev @mjura
+
+## Context
+
+As an operator, I want the choice to deploy Rancher Turtles in the same or a different cluster to Rancher Manager, so that I have choice in my deployment topology.
+
+The initial implementation of Rancher Turtles assumes that the operator runs in the same cluster as Rancher Manager.
```

Comment:
I think this could be removed. We are now making the initial version of turtles, and there is no previous ADR on this topic to replace.